pkg/autoscaler/aggregation/bucketing.go

```diff
@@ -87,6 +108,49 @@ func roundToNDigits(n int, f float64) float64 {
 	return math.Floor(f*p) / p
 }
 
+const (
+	// minExponent is the minimal decay multiplier for the weighted average computations.
+	minExponent = 0.2
+	// The sum of weights for weighted average should be at least this much.
+	weightPrecision = 0.9999
```

Comment:
Which of these are we expecting a user to eventually be able to set? Or are we planning to fiddle until we find one value that we like?

Reply:
I can expect `minExponent` _may be_ settable. For now neither :)